torch_frame/datasets/fake.py

```diff
@@ -80,6 +80,16 @@ def __init__(
                 if with_nan:
                     df_dict[col_name][0] = None
                 col_to_stype[col_name] = stype.multicategorical
+        if stype.sequence_numerical in stypes:
+            for col_name in ['seq_1', 'seq_2']:
+                arr = []
+                for i in range(num_rows):
+                    list_len = np.random.randint(1, 3)
+                    arr.append(np.random.randint(0, 3, size=list_len).tolist())
```

Comment:
super nit: I feel you don't need `numpy` here.
Just use pure Python. something like
```python
sequence_length = random.randint(5, 20)
sequence = [random.randint(1, 100) for _ in range(sequence_length)]
```